### PR TITLE
Update cadvisor godep to v0.27.3

### PR DESCRIPTION
**Pull request by dashpole**
**What this PR does / why we need it**:
Update cadvisor version from v0.27.2 to v0.27.3.

**Special notes for your reviewer**:
cAdvisor keeps one branch for each cadvisor branch.  The v0.27 branch tracks the kubernetes 1.8 branch.

**Release note**:

```release-note
Update jquery and bootstrap dependencies
```
/assign @tallclair

### CODE REVIEW

File: vendor/github.com/vishvananda/netns/BUILD [hunk 1]
```diff
@@ -1,11 +1,4 @@
-package(default_visibility = ["//visibility:public"])
-
-licenses(["notice"])
-
-load(
-    "@io_bazel_rules_go//go:def.bzl",
-    "go_library",
-)
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```
Comment (tallclair) [inline vendor/github.com/vishvananda/netns/BUILD]: Revert this file?

Comment (dashpole) [reply vendor/github.com/vishvananda/netns/BUILD]: done

**Comment by tallclair** [comment]
/assign @jpbetz 

For cherry-pick approval.

**Pull request merged**